tests/core/editable/manual/enterbr.md

```diff
@@ -0,0 +1,20 @@
+@bender-tags: 4.16.3, bug, 3858
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, clipboard, enterkey, format,
+
+
+1. Open browser's console.
+2. Copy two paragraphs placed above the editor. `<p>a</p><p>b</p>` and paste into editable.
+3. Hit enter twice.
+4. Type any ( only one ) letter and remove it using `backspace`.
+5. Paste the the same two paragraphs from `step 1` ( don't change cursor position ).
+
+**Expected**
```

Comment:
According to our [code style guide](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#31-manual-tests) there should be an empty lines after `Expected` and `Unexpected`. Please add them.